packages/bcer-retailer-app/app/src/components/generic/RequiredFieldLabel.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+
+export default function RequiredFieldLabel ({label}: {label: string}) {
+    return (

```

Comment:
Looks like there's a bit extra spacing here compared to the other files. Tabs -> 2 spaces?